_events/OSG-User-School-2023.md

```diff
@@ -0,0 +1,57 @@
+---
+short_title: OSG User School 2023
+title: "OSG User School 2023, Aug. 7–11"
+layout: events
+published: true
+
+excerpt: |
+    Applications for the OSG User School 2023 at the University of Wisconsin-Madison are now open!
+start_date: 2023-08-07
+end_date: 2023-08-11
+location: "University of Wisconsin-Madison"
+link: "https://osg-htc.org/user-school-2023/"
+image: "/images/osg-user-school-2023-event.jpg"
+header_image: "/images/osg-user-school-2023-event.jpg"
+---
+
+{% capture main %}
+
+The OSG User School provides researchers the opportunity to learn how to use high-throughput computing (HTC) systems to run large-scale computing applications at their campus or using the national-scale OSG consortium. Please join us and learn how you can use HTC methods and services to handle large-scale computing applications to advance your research!
+
+To learn more about the event, check out these articles written about the OSG User School 2022:
+- Read about last year's [OSG User School student lightning talks](https://osg-htc.org/spotlights/Lightning-Talks.html), where researchers showcased their work and future plans to implement HTC.
+- This article spotlights [studens from the NIAID/ACE students who attended this year’s OSG User School 2022](https://osg-htc.org/spotlights/NIAID-ACE-students-attend-OSG-User-School.html). Learn more about these students and their work and the ongoing collaboration between the OSG and ACE/NIAID teams.
+- To view last year's lesson plans and materials, check out the [OSG User School Concludes](https://osg-htc.org/spotlights/OSG-User-School-Concludes.html) article on the OSG website.
+
```

Comment:
Add:

“
<span style="font-size: larger; color: #FF6600;">Apply by April 17 on [the OSG School website](https://osg-htc.org/user-school-2023/).</span>
”

[Yes, it repeats the stuff on the right, but I’m fine with this here.]